### PR TITLE
Add objectFit='cover' to various Landscape avatars to avoid aspect ratio squishing

### PR DESCRIPTION
**Pull request by 0x70b1a5**
**PLEASE NOTE** I was unable to test this PR locally as I had issues cloning the repository (something with `con.c`, notable that I initially attempted to clone via Windows, fool that I am!).

Anyway, shout out to @matildepark for telling me where to change the things that needed to change. You may also wish to squash these commits as they were made via GitHub web (due to aforesaid git clone issue).

### CODE REVIEW

**Review by matildepark — CHANGES_REQUESTED** [review]
This doesn't work — you'll need `style={{ objectFit: 'cover' }}`; also use `display="inline-block"` not dib class.

**Comment by matildepark** [comment]
So yeah, in sum -

- the style change previously specified
- in one commit
- labelled something like `landscape: correct avatar aspect ratio`

You did this work off your fork's master, which might make this difficult, though.

**Comment by 0x70b1a5** [comment]
Thanks! I'll try booting up my Linux machine and working on this there, to catch anything else that crops up.

**Comment by matildepark** [comment]
You'll want to squash this to one commit.

```
git rebase -i 6bcbbf8f1a4756c195a324efcf9515b6f288f700
```

Then change all the commits in the list to `squash`, but the first commit to `reword`. Reword it to `landscape: correct avatar aspect ratio` and then save, `git push -f`.

**Comment by 0x70b1a5** [comment]
Oh gotcha, I had misunderstood what "in one commit" meant. Apologies.

File: pkg/interface/src/views/landscape/components/Participants.tsx [hunk 1]
```diff
@@ -297,7 +297,13 @@ function Participant(props: {
 
   const avatar =
     contact?.avatar !== null && !hideAvatars ? (
-      <img src={contact.avatar} height={32} width={32} className="dib" />
+      <img
```
Comment (matildepark) [inline pkg/interface/src/views/landscape/components/Participants.tsx:300]: `display="inline-block"` actually won't work with the unstyled `img` element. What you'll want is to add an `Image` import from indigo-react (you can find the indigo elements we import at the top), then just change this to `<Image`. That's all. You can append it as a review comment — `landscape: address #4517 feedback`

**Review by matildepark — CHANGES_REQUESTED** [review]

File: pkg/interface/src/views/landscape/components/Participants.tsx (hunk 1 above)
Comment (0x70b1a5) [reply pkg/interface/src/views/landscape/components/Participants.tsx]: Thank you for your patience.

**Review by matildepark — APPROVED** [review]
